src/unix/bsd/apple/mod.rs

```diff
@@ -3297,6 +3297,10 @@ extern "C" {
     pub fn fchflags(fd: ::c_int, flags: ::c_uint) -> ::c_int;
     pub fn clock_getres(clk_id: ::clockid_t, tp: *mut ::timespec) -> ::c_int;
     pub fn clock_gettime(clk_id: ::clockid_t, tp: *mut ::timespec) -> ::c_int;
+    pub fn clock_getcpuclockid(
+        pid: ::pid_t,
+        clk_id: *mut ::clockid_t,
+    ) -> ::c_int;
```

Comment:
I couldn't find any reference for `clock_getcpuclockid` on macOS. Did you confirm if it exists?

Reply:
Indeed, looks like macOS doesn't support it. Removed.